app/src/main/res/layout/fragment_crash_reporter.xml

```diff
@@ -18,7 +19,8 @@
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintWidth_percent="0.8"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent"/>
+        app:layout_constraintTop_toTopOf="parent"
+        tools:ignore="ContentDescription" />
```

Comment:
I think it's technically better to set importantForAccessibility="no".

Reply:
Android Studio has been misleading me this whole time!